### PR TITLE
First deploy attempt

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Try to deploy VM automatically

---

**Related issue :**

### CODE REVIEW

File: playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  roles:
+  - role: 'grycap.docker'
+
+- hosts: localhost
+  connection: local
+  gather_facts: yes
+  tasks:
+  - name: Get dashboard repo
+    ansible.builtin.git:
+      repo: 'https://github.com/enolfc/horizon-aggregator.git'
```
Comment (sebastian-luna-valero) [inline playbook.yaml:14]: Should this go somewhere under https://github.com/EGI-Federation/?

File: playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        RemainAfterExit=true
+        WorkingDirectory=/horizon-aggregator
+        ExecStart=/usr/local/bin/docker-compose up -d --remove-orphans
+        ExecStop=/usr/local/bin/docker-compose down
+
+        [Install]
+        WantedBy=multi-user.target
+      dest: /etc/systemd/system/dashboard.service
+
+  - name: Run, baby run
```
Comment (sebastian-luna-valero) [inline playbook.yaml:46]: LOL

File: site-config.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            "$(yq -r .clouds.backend.site config.yaml)" \
+            "$(yq -r .clouds.backend.vo config.yaml)" \
+            "$OIDC_TOKEN" >> tmp-clouds.yaml
+
+dump_config deploy \
+            "$(yq -r .clouds.deploy.site config.yaml)" \
+            "$(yq -r .clouds.deploy.vo config.yaml)" \
+            "$OIDC_TOKEN" >> tmp-clouds.yaml
+
+mv tmp-clouds.yaml clouds.yaml
```
Comment (sebastian-luna-valero) [inline site-config.sh:46]: Why not `dump_config [etc.] >> clouds.yaml` directly?

File: configure.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ansible-galaxy install grycap.docker
+
+if ansible-playbook playbook.yaml >ansible.log 2>&1 ; then
+   status_summary="success"
+   #color="#6DBF59"
+   #header="Successful deployment :rocket:"
+else
+   status_summary="fail"
+   #color="#EA4F47"
+   #header="Failed deployment :boom:"
```
Comment (sebastian-luna-valero) [inline configure.sh:17]: Having this back should be high priority :D

File: playbook.yaml (hunk 1 above)
Comment (enolfc) [reply playbook.yaml:14]: yes, or even just have one repo if we feel more appropriate

File: site-config.sh (hunk 3 above)
Comment (enolfc) [reply site-config.sh:46]: the `fedcloudclient openstack` command above gets crazy with a half-baked `clouds.yaml` file

**Comment by enolfc** [comment]
I'm merging, let's see if it works. And we move from here with gradual improvements

**Pull request merged**